sqrl-testing/sqrl-integration-tests/src/test/resources/snapshots/com/datasqrl/UseCaseCompileTest/postgres-log-gql-postgres-log-gql-package.txt

```diff
@@ -0,0 +1,5 @@
+[FATAL] Timestamp column not found: "event_time". Must be one of: [_uuid, eventType, timestamp, userId]
```

Comment:
This shouldn't error, right?

Reply:
This test was initially created when we didn't have the 'create table' and 'export to log' statements. I intended to keep it because it would be useful for the Vertx component of this task, with plans to fix it during that phase. However, I agree with your point about not committing a failing test, so I've removed it.